tests/spec_tests/query_tests/union.yml

```diff
@@ -63,15 +44,6 @@ tests:
       - {'': {'a': 2}}
       - {'': {'a': 2}}
 
-  - description: union is left associative
-    current_db: mydb
-    query: "SELECT VALUE {'a': a} FROM foo UNION ALL SELECT VALUE {'a': a} FROM foo UNION SELECT VALUE {'a': a} FROM foo"
-    ordered: false
-    skip_reason: "distinct UNION not currently supported"
```

Comment:
Not sure why this one was skipped since it's doing a `UNION ALL`. Should we leave this one here? When you cut and pasted it into the other file, you left it as `UNION ALL` so it's out of place in the `distinct.yml` file.

Reply:
Hm, there's a UNION distinct on the right side to show that it is executing the `UNION ALL` on the left first to duplicate `foo` and then the UNION distinct will deduplicate it and so a:1 and a:2 only appears once.  Double checking if you missed the second UNION?  Otherwise sure I can move it back here in `union.yml`.